src/topology/algebra/order/basic.lean

```diff
@@ -2188,6 +2188,18 @@ lemma is_compact.bdd_above {α : Type u} [topological_space α] [linear_order α
   [order_closed_topology α] : Π [nonempty α] {s : set α}, is_compact s → bdd_above s :=
 @is_compact.bdd_below (order_dual α) _ _ _
 
+/-- A continuous function is bounded below on a compact set. -/
+lemma is_compact.bdd_below_image {α : Type u} [topological_space α] [linear_order α]
```

Comment:
I understand why you put these here, but I think they might be better with the other things related to the extreme value theorem in `topology.algebra.order.compact` (for example, near `is_compact.exists_forall_le`). Although this is not a strong preference.

Reply:
I can take a look what will break if `is_compact.bdd_[below|above]` are moved to `topology.algebra.order.compact`

Reply:
In retrospect I think it's probably better to leave those lemmas where they are. They are related to the extreme value theorem, but much weaker, so I think it's fine to put them where they are.